app/src/main/java/com/kickstarter/viewmodels/MessageThreadsViewModel.java

```diff
@@ -77,8 +77,10 @@ public ViewModel(final @NonNull Environment environment) {
       this.client = environment.apiClient();
       this.currentUser = environment.currentUser();
 
+      final Observable<Void> startOverWith = Observable.merge(this.onResume, this.refresh);
```

Comment:
for posterity `startOverWith` is a signal that we typically use for our paginators that indicate when we should fetch new data